configuration/multimedia.md

```diff
@@ -386,3 +384,109 @@ listenAndAnswer(null, null)
+But the `turn off tv` phase now detect a collision because two matching items accept the `OFF` command.
+
+#### Name prevalence
+
+One way you can solve this is by using the name prevalence, items with start with other items names take prevalence over them.
+
+If the Switch item has the name `tv` and the and Dimmer item the name `tv_brightness` there will be no collisions between them and the `OFF` command will target the Switch item.
+
+#### Exact match label/synonym prevalence
+
+Other way you can solve this is by using the exact match prevalence, items with have an exact match on their label/synonyms take prevalence.
+
+If the Switch item has the synonym `TV` there will be no collisions between them and the `OFF` command will target the Switch item.
+
+#### Location prevalence
+
+The dialog processor forward its locationItem to the standard interpreter to be used for reducing collisions on the target resolution.
+
+If you have two items labeled as `Light` but one is a child of the location item you have configured for the dialog execution, the item takes prevalence.
+
+The location takes prevalence over an exact match.
+
+#### Item description rules
+
+The interpreter also creates rules for your item descriptions for English, German, French, Italian and Spanish.
+
+If you have a Dimmer item called `Light` with command description `100=high` the interpreter will also understand the phrase `Set light to high`.
+
+### Item custom rules.
+
+You can register custom rules into the interpreter using items and the metadata namespace `voiceSystem`.
+
+The examples of valid rules:
+
+```
+"start? watch|watching $*$ on $name$":
+Matches "start watching some show on tv" sends command "some show".
+
+"watch|play $*$ on tv":
+Matches "play some show on tv" sends command "some show".
+
+"watch|play $cmd$":
+Matches "play some show" and sends command "some_show_id" only if the item command description contains `some_show_id=some show`.
+
+"start? watch|watching $cmd$ at|on? $name$":
+Matches "watch some show tv" only if item command description contains `some_show_id=some show`, sends command "some_show_id".
+```
+
+As you can see there are some reserved tokens and characters:
+
+* `$name$` defines the place of the item name (resolved as explained before), is optional.
+* `$cmd$` defines the place of a command label, extracted from the item command description.
+* `$*$` defines the place of a command, whose value is not constrained.
+* `|` defines alternative word tokens.
+* `?` defines optional word token.
+
+The item metadata namespace `voiceSystem` allows these configurations:
+
+* **isTemplate**: The rule defined on these item metadata will not target it but similar items (items with same tags and semantic).
+* **isSilent**: The interpreter will say nothing in case these rule is executed correctly (util if you plan on trigger a rule and talk from it).
```

Comment:
in case the rule is executed

>(util if you plan on trigger a rule and talk from it).

Do you mean?
> (until you implement and trigger a rule and talk from within that rule)

Reply:
No, what I meant is that the situation the option was added for is when you want to handle the command from a rule and talk from there. 
Because it's weird that the interpreter says "Ok" if you are using the target item to trigger a rule and do extra work there.

An example will be having a command "Search in google $*$" writing to a String Item that you will use to trigger a rule that search programmatically at google and told you something.

Reply:
I can change it to :
```
(an possible use case can be trigger a rule on Item command to answer the command programmatically)
```

Reply:
"a possible use case can be trigger a rule on Item command to answer programmatically"

-> a possible use case can be a trigger for a rule on an Item command to answer programmatically